src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -78,6 +83,9 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         }
 
         CommandResult commandResult;
+        if (isYes(commandText)) {
+            commandText = "yes";
```

Comment:
Minor: I'm not sure if this violates the coding quality standard of not reusing formal parameters, but I'm also not sure what's an alternative tho HAHA

Reply:
O maybe it can be 
```suggestion
    Command command;
    if (isYes(commandText)) {
        command = gradPadParser.parseCommand("yes");
    } else {
        command = gradPadParser.parseCommand(commandText);
    }       
```